packages/amplication-client/src/VersionControl/PendingChange.tsx

```diff
@@ -26,10 +26,11 @@ const ACTION_TO_LABEL: {
 const PendingChange = ({ change, linkToOrigin = false }: Props) => {
   const { currentWorkspace, currentProject } = useContext(AppContext);
   /**@todo: update the url for other types of blocks  */
-  const url =
-    change.originType === models.EnumPendingChangeOriginType.Entity
-      ? `/${currentWorkspace?.id}/${currentProject?.id}/${change.resource.id}/entities/${change.originId}`
-      : `/${currentWorkspace?.id}/${currentProject?.id}/${change.resource.id}/update`;
+  const baseUrl = `/${currentWorkspace?.id}/${currentProject?.id}/${change.resource.id}`;
+  // const url =
+  //   change.originType === models.EnumPendingChangeOriginType.Entity
+  //     ? `/${currentWorkspace?.id}/${currentProject?.id}/${change.resource.id}/entities/${change.originId}`
+  //     : `/${currentWorkspace?.id}/${currentProject?.id}/${change.resource.id}/settings/update`;
```

Comment:
Let remove the comment

Reply:
Thanks Ofek, we have another task that will change the entire component and Mor is on it